src/PasswordGenerator.php

```diff
@@ -174,9 +175,13 @@ public static function EN(): PasswordGenerator
      * @static
      * @return PasswordGenerator configured instance that uses cache only
      */
-    public static function CACHED(): PasswordGenerator
+    public static function CACHED(?string $wordCacheFile): PasswordGenerator
```

Comment:
I think it would be better to use default param instead to keep backward compatibility:
```suggestion
    public static function CACHED(string $wordCacheFile = null): PasswordGenerator
```